src/API.jl

```diff
@@ -989,7 +989,7 @@ function precompile(ctx::Context; internal_call::Bool=false, kwargs...)
     num_tasks = parse(Int, get(ENV, "JULIA_NUM_PRECOMPILE_TASKS", string(Sys.CPU_THREADS::Int + 1)))
     parallel_limiter = Base.Semaphore(num_tasks)
     io = ctx.io
-    fancyprint = can_fancyprint(io)
+    fancyprint = can_fancyprint(io) && !debugmode
```

Comment:
Ah, this still needs switching when in debug mode. Is there a way to determine that, other than checking `JULIA_DEBUG`? @fredrikekre

Reply:
just fixed this. Not quite perfect as it will print the following, which is a little confusing. Is there a better option?
```
┌ Debug: false
└ @ Pkg.API ~/Documents/GitHub/Pkg.jl/src/API.jl:987
```